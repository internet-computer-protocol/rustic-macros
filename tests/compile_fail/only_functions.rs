use rustic_macros::modifiers;

#[modifiers("main")]
struct S;

fn main() {}