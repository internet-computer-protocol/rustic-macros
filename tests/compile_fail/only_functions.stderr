error: expected `fn`
 --> tests/compile_fail/only_functions.rs:4:1
  |
4 | struct S;
  | ^^^^^^
